repositories/analytics_views/apikeys.sql

```diff
@@ -0,0 +1,10 @@
+DROP VIEW IF EXISTS analytics.apikeys;
+
+CREATE VIEW analytics.apikeys WITH (security_invoker = false) AS
+SELECT
```

Comment:
note that private fields that should never leave our prod environment, such as in this case the api key value, are not exposed in the view